pkg/util/fedinformer/typedmanager/single-cluster-manager.go

```diff
@@ -69,15 +69,17 @@ type SingleClusterInformerManager interface {
 
 // NewSingleClusterInformerManager constructs a new instance of singleClusterInformerManagerImpl.
 // defaultResync with value '0' means no re-sync.
-func NewSingleClusterInformerManager(client kubernetes.Interface, defaultResync time.Duration, parentCh <-chan struct{}) SingleClusterInformerManager {
+func NewSingleClusterInformerManager(client kubernetes.Interface, defaultResync time.Duration, parentCh <-chan struct{}, transformFuncs map[schema.GroupVersionResource]cache.TransformFunc) SingleClusterInformerManager {
 	ctx, cancel := util.ContextForChannel(parentCh)
 	return &singleClusterInformerManagerImpl{
-		informerFactory: informers.NewSharedInformerFactory(client, defaultResync),
-		handlers:        make(map[schema.GroupVersionResource][]cache.ResourceEventHandler),
-		syncedInformers: make(map[schema.GroupVersionResource]struct{}),
-		ctx:             ctx,
-		cancel:          cancel,
-		client:          client,
+		informerFactory:  informers.NewSharedInformerFactory(client, defaultResync),
+		handlers:         make(map[schema.GroupVersionResource][]cache.ResourceEventHandler),
+		syncedInformers:  make(map[schema.GroupVersionResource]struct{}),
+		startedInformers: make(map[schema.GroupVersionResource]bool),
```

Comment:
It's ambiguous between the field name(`startedInformers`) and its type.
From the name `startedInformers` I can get this is the map for storing `started` informers only, but why it's type allows a `bool`?